intellij-toml/core/src/test/kotlin/org/toml/ide/inspections/TomlInspectionsTestBase.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.toml.ide.inspections
+
+import com.intellij.codeInspection.InspectionProfileEntry
+import org.toml.ide.annotator.TomlAnnotationTestBase
+import org.toml.ide.annotator.TomlAnnotationTestFixture
+import kotlin.reflect.KClass
+
+abstract class TomlInspectionsTestBase(
```

Comment:
Let's add a test for documentations existence, like in `testInspectionHasDocumentation`